config-linux.md

```diff
@@ -494,6 +494,21 @@ You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given ent
 }
 ```
 
+## <a name="configLinuxUnified" />Unified
+
+**`unified`** (object, OPTIONAL) allows cgroup v2 parameters to be modified at runtime for the container.
+
+Each key in the map refers to a file in the cgroup unified hierarchy.  The OCI runtime must ensure that the needed cgroup controllers are enabled for the cgroup.
```

Comment:
Missing line break at the end of the sentence (I don't like this rule either, but :shrug:).

Reply:
Also, MUST should be capitalised. This should also have some language about how to deal with controllers that are unsupported by the system (or the runtime) -- but also that unknown-to-the-runtime configuration options for known-to-the-runtime controllers SHOULD (or maybe even MUST?) be written to the relevant cgroup file.

And I guess this means we're punting on the whole cgroupv1 conversion issue? Or do you want to tackle that in a separate PR?

Reply:
I can address it here.

Do we want the conversion to be mandatory?

I think it could be suggested, but I'd not put too much requirements on this area, something like:
```
If a controller is enabled on cgroup v2 and the configuration is provided
for cgroup v1, the runtime COULD attempt a conversion.
If the conversion is not possible the runtime MUST generate an error.

/* Some examples */
```

Also to keep in mind, an OCI runtime could support hybrid mode, so the cgroup v1 configuration still makes sense.

Reply:
thanks, fixed